pulumi-awsx fails NPM validation

### What happened?

When I install `pulumi-awsx` locally and run `npm test --workspaces` I get the following error:

```
$ npm test --workspaces

> @charm-tx/pulumi-aws@0.2.1 test
> tsc --noEmit

../node_modules/@pulumi/awsx/classic/ec2/vpcTopology.d.ts:4:174 - error TS2304: Cannot find name 'AvailabilityZoneDescription'.

4 export declare function create(resource: pulumi.Resource | undefined, vpcName: string, vpcCidr: string, ipv6CidrBlock: pulumi.Output<string> | undefined, availabilityZones: AvailabilityZoneDescription[], numberOfNatGateways: number, assignGeneratedIpv6CidrBlock: pulumi.Input<boolean>, subnetArgsArray: VpcSubnetArgs[]): VpcTopologyDescription;
                                                                                                                                                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in ../node_modules/@pulumi/awsx/classic/ec2/vpcTopology.d.ts:4
```

### Steps to reproduce

Run with the following `tsconfig.json`:

```
{
  "compilerOptions": {
    "target": "es2022",
    "module": "esnext",
    "moduleResolution": "node16",
    "resolveJsonModule": true,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "noFallthroughCasesInSwitch": true,
    "noImplicitReturns": true,
    "forceConsistentCasingInFileNames": true,
    "declaration": true,
    "sourceMap": true,
    "pretty": true
  },
  "exclude": ["node_modules"]
}

```

and the `package.json`
```
{
  "name": "<name>",
  "version": "0.2.1",
  "type": "module",
  "scripts": {
    "prepublishOnly": "tsc",
    "test": "tsc --noEmit"
  },
  "module": "./lib/index.js",
  "files": [
    "lib/"
  ],
  "exports": {
    ".": "./lib/index.js",
    "./*": "./lib/*.js"
  },
  "typesVersion": {
    "*": {
      "*": [
        "./lib/*"
      ]
    }
  },
  "dependencies": {
    "@pulumi/aws": "^5.16.0",
    "@pulumi/awsx": "1.0.0-beta.7",
    "@pulumi/docker": "^3.4.1",
    "zod": "^3.19.1"
  }
}
```

### Expected Behavior

The test should pass

### Actual Behavior

The test fails

### Output of `pulumi about`

_No response_

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -186,9 +186,10 @@ jobs:
           name: ${{ matrix.language  }}-sdk.tar.gz
           path: ${{ github.workspace}}/sdk/${{ matrix.language }}.tar.gz
       - name: Install gotestfmt
-        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        uses: GoTestTools/gotestfmt-action@v2
         with:
-          repo: haveyoudebuggedit/gotestfmt
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version: v2.4.0
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:

--- a/.github/workflows/master.yml
+++ b/.github/workflows/master.yml
@@ -185,9 +185,10 @@ jobs:
           name: ${{ matrix.language  }}-sdk.tar.gz
           path: ${{ github.workspace}}/sdk/${{ matrix.language }}.tar.gz
       - name: Install gotestfmt
-        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        uses: GoTestTools/gotestfmt-action@v2
         with:
-          repo: haveyoudebuggedit/gotestfmt
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version: v2.4.0
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:

--- a/.github/workflows/pre-release.yml
+++ b/.github/workflows/pre-release.yml
@@ -185,9 +185,10 @@ jobs:
           name: ${{ matrix.language  }}-sdk.tar.gz
           path: ${{ github.workspace}}/sdk/${{ matrix.language }}.tar.gz
       - name: Install gotestfmt
-        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        uses: GoTestTools/gotestfmt-action@v2
         with:
-          repo: haveyoudebuggedit/gotestfmt
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version: v2.4.0
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -184,9 +184,10 @@ jobs:
           name: ${{ matrix.language  }}-sdk.tar.gz
           path: ${{ github.workspace}}/sdk/${{ matrix.language }}.tar.gz
       - name: Install gotestfmt
-        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        uses: GoTestTools/gotestfmt-action@v2
         with:
-          repo: haveyoudebuggedit/gotestfmt
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version: v2.4.0
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:

--- a/.github/workflows/run-acceptance-tests.yml
+++ b/.github/workflows/run-acceptance-tests.yml
@@ -211,9 +211,10 @@ jobs:
           name: ${{ matrix.language  }}-sdk.tar.gz
           path: ${{ github.workspace}}/sdk/${{ matrix.language }}.tar.gz
       - name: Install gotestfmt
-        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        uses: GoTestTools/gotestfmt-action@v2
         with:
-          repo: haveyoudebuggedit/gotestfmt
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version: v2.4.0
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:

--- a/awsx-classic/ec2/vpcTopology.ts
+++ b/awsx-classic/ec2/vpcTopology.ts
@@ -21,7 +21,6 @@ import { Cidr32Block, getIPv4Address } from "./cidr";
 
 import * as utils from "../utils";
 
-/** @internal */
 export interface AvailabilityZoneDescription {
     name: string;
     id: string;

--- a/sdk/nodejs/classic/ec2/vpcTopology.ts
+++ b/sdk/nodejs/classic/ec2/vpcTopology.ts
@@ -21,7 +21,6 @@ import { Cidr32Block, getIPv4Address } from "./cidr";
 
 import * as utils from "../utils";
 
-/** @internal */
 export interface AvailabilityZoneDescription {
     name: string;
     id: string;

